user-guide/Reference/Feature_overview.md

```diff
@@ -0,0 +1,95 @@
+---
+uid: Feature_overview
+---
+
+# Feature overview
+
+## Analytics
+
+|Feature | Minimum DataMiner version | Soft-launch option | License | Database |
```

Comment:
We should change the headers like this:
```suggestion
|Feature | Minimum DataMiner version | Minimum soft-launch version | Notes |
```
"Minimum DataMiner version" should contain "N/A" if the feature is only available in soft launch.